frontend/apps/core-authnz-frontend/src/components/navbar/NavBar.tsx

```diff
@@ -22,12 +22,12 @@ export const NavBar: FC = (props) => {
         <div className="collapse navbar-collapse" id="navbarNav">
           <ul className="navbar-nav">
             <li className="nav-item">
-              <NavLink className="nav-link" activeClassName="active" to="/organizations">
+              <NavLink className="nav-link" to="/organizations">
```

Comment:
Found it.
```
<NavLink
            to="tasks"
            className={({ isActive }) =>
              isActive ? activeClassName : undefined
            }
          >
```

Reply:
according to the docs, an `active` class will be applied by default when active, and since this is the class we've been using, I didn't see the need to add it ourselves.